### ISSUE TITLE
Widget is gone when config is added

### ISSUE THREAD

**Issue by janzenz** [fightforthefuture/digital-climate-strike#88]
Hi I've noticed that the widget is hidden once I add `DIGITAL_CLIMATE_STRIKE_OPTIONS` in the page, I used the following snippet below:

```html
<script type="text/javascript">
            var DIGITAL_CLIMATE_STRIKE_OPTIONS = {
                disableGoogleAnalytics: true,
                footerDisplayStartDate: new Date(2019, 9, 27),
            }
</script>
<script src="https://assets.digitalclimatestrike.net/widget.js" async></script>
```

**Comment by dtoakley** [comment]
Hi @janzenz -- this is because you have set the the widget to start displaying on on October 27th, so it won't show until that date. If you remove footerDisplayStartDate form that config object, it will show.

**Comment by janzenz** [comment]
@dtoakley thank you for getting back so quickly and that's a good catch. I thought if you use `#ALWAYS_SHOW_DIGITAL_CLIMATE_STRIKE` at the end of the URL forces to show the widget regardless of the date, I guess I'm wrong.

**Comment by dtoakley** [comment]
Well, you'd ID'd an issue with the usage of that hash, which is great. I'll add an issue for fixing that, but not sure if we'll be able to do it this week. Thanks @janzenz

**Comment by janzenz** [comment]
@dtoakley here's my attempt for the fix #93

**Comment by dtoakley** [comment]
Great, I'll review that tomorrow. Going to close this issue as it relates to something else :)
